src/main/java/com/hartwig/snpcheck/SnpCheck.java

```diff
@@ -57,7 +57,8 @@ public SnpCheck(final RunApi runs, final SampleApi samples, final Bucket snpchec
     public void handle(final PipelineStaged event) {
         if (event.analysisType().equals(Type.TERTIARY) && event.analysisContext().equals(Context.DIAGNOSTIC)) {
             Run run = runs.get(event.runId().orElseThrow());
-            if (run.getIni().equals(Ini.SOMATIC_INI.getValue()) || run.getIni().equals(Ini.SINGLESAMPLE_INI.getValue())) {
+            if (run.getStatus().equals(Status.FINISHED) &&
```

Comment:
this reads like finished and somatic OR single

Reply:
Good call! Updated.